code/datums/diseases/advance/symptoms/sneeze.dm

```diff
@@ -36,4 +36,6 @@ Bonus
 			else
 				M.emote("sneeze")
 				A.spread(A.holder, 5)
+				M.Stun(1)

```

Comment:
Just for your information...`Stun()` still causes you to drop the item in your active hand.